config/settings/base.py

```diff
@@ -113,7 +113,7 @@
 
 LANGUAGE_CODE = 'en-us'
 
-TIME_ZONE = 'UTC'
+TIME_ZONE = 'America/Phoenix'
```

Comment:
It would affect our dates though, wouldn't it? Say we post a workshop (like the short example above) that's from 8/11 to 8/12 in Melbourne, Australia. On our site that would still be open to everyone around the planet because it's only 8/10 here, but it's 9AM 8/11 in Melbourne, so it should be closed and already started.